samples/drivers/misc/morse_code/README.rst

```diff
@@ -0,0 +1,54 @@
+.. zephyr:code-sample:: morse_code
+   :name: Morse Code
+   :relevant-api: morse_code
+
+   Send a text using Morse Code with a LED.
+
+Overview
+********
+
+In this sample a text can be send using Morse Code driver. The Morse Code is
+implemented based on the `M.1677-International Morse code`_ norm.
+
+When the application starts it automatically sends the text "PARIS". This word
+is universally used to determine the number of Words Per Minute (wps). This
+means that when using a speed of 20 it is possible to transmit the word "PARIS"
+20 times in a minute.
+
+Requirements
+************
+
+To use this sample, the following hardware is required:
+
+* A board with a LED
+* A Smartphone with an application like `Morse Code Engineer`_.
+
+Building and Running
+********************
+
+This sample should work on any board that has PWM enabled. For example, it can
+be run on the ``sam_v71_xult`` board as described below:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/drivers/misc/morse_code
+   :board: sam_v71_xult
+   :goals: flash
+   :compact:
+
+Sample Output
+=============
+
+.. code-block:: console
```

Comment:
I think you should add a description of the expected "real-life" behavior one will observe on the device, in addition to providing the shell interaction that follows.

Reply:
I added a few more info, please review.
I changed the code to use GPIO instead of LED, just to mention.